packages/release/Cargo.toml

```diff
@@ -35,12 +35,9 @@ kexec-tools = { path = "../../packages/kexec-tools" }
 libaudit = { path = "../libaudit" }
 libgcc = { path = "../libgcc" }
 libstd-rust = { path = "../libstd-rust" }
+log4j2-hotpatch= { path = "../log4j2-hotpatch" }
```

Comment:
nit: this seems like more properly a dependency of `hotdog` which is what consumes the hot patch jar